Weather App/UI/SearchVC/SearchViewController.swift

```diff
@@ -65,13 +65,15 @@ extension SearchViewController {
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
          let cell = tableView.dequeueReusableCell(withIdentifier: "cell",
                                                   for: indexPath)
+         let cellFontSize: CGFloat = UIScreen.main.bounds.height > 600 ? 30 : 24
```

Comment:
Could you do the design/setup of the cell in a different place? Can the cell iteself have a setup method for its setup?

Reply:
Yes, I can move this designing/setup into a method of the cell class itself, I will move it.